docs/book/src/clusterctl/developers.md

```diff
@@ -91,6 +92,21 @@ clusterctl init \
 
 ```
 
+Check the components are running properly:
```

Comment:
This check is right after the `clusterctl init` so we would expect the kind cluster is created.

Reply:
took a second look, you are right - the `clusterctl init` is just an output from the hack script. I will see if there is any benefit by moving this to the quick start. (It would have helped me debug the webhook issue if this information was there)